src/i18n/locales/en/login.json

```
@@ -0,0 +1,4 @@
+{
+    "Logging In...": "Logging In...",
```

Nit: Can we try to avoid using dots in keys to avoid confusion with nested keys?


```suggestion
    "Logging In": "Logging In...",
```